lib/msal-core/src/telemetry/UiEvent.ts

```diff
@@ -0,0 +1,19 @@
+import TelemetryEvent from "./TelemetryEvent";
+import { EVENT_NAME_PREFIX } from "./TelemetryConstants";
+
+const USER_CANCELLED = EVENT_NAME_PREFIX + "user_cancelled";
+const ACCESS_DENIED = EVENT_NAME_PREFIX + "access_denied";
+
+export default class UiEvent extends TelemetryEvent {
```

Comment:
Would popup window and iframe events be part of this class?

Reply:
Potentially, those are not in any other telem examples, but as we instrument, we can add to it

Reply:
I think we should probably try to add those sooner rather than later, since UI related issues are common.

Reply:
Will add as we start to instrument the apis